SampleApplications/Samples/NetCoreConsoleServer/Program.cs

```diff
@@ -248,22 +250,43 @@ private async Task ConsoleSampleServer()
             }
 
             // start the server.
-            server = new SampleServer();
-            await application.Start(server);
+            Server = new SampleServer();
+            await application.Start(Server);
+
+            if (ReverseConnectUrl != null)
+            {
+                Server.AddReverseConnection(ReverseConnectUrl);
+            }
+
+            // print reverse connect info
+            Console.WriteLine("Reverse Connect Clients:");
+            var reverseConnections = Server.GetReverseConnections();
+            foreach (var connection in reverseConnections)
+            {
+                Console.WriteLine(connection.Key);
+            }
+
+            // print endpoint info
+            Console.WriteLine("Server Endpoints:");
+            var endpoints = Server.GetEndpoints().Select(e => e.EndpointUrl).Distinct();
+            foreach (var endpoint in endpoints)
+            {
+                Console.WriteLine(endpoint);
+            }
 
             // start the status thread
-            status = Task.Run(new Action(StatusThread));
+            Status = Task.Run(new Action(StatusThread));
 
             // print notification on session events
-            server.CurrentInstance.SessionManager.SessionActivated += EventStatus;
-            server.CurrentInstance.SessionManager.SessionClosing += EventStatus;
-            server.CurrentInstance.SessionManager.SessionCreated += EventStatus;
+            Server.CurrentInstance.SessionManager.SessionActivated += EventStatus;
+            Server.CurrentInstance.SessionManager.SessionClosing += EventStatus;
+            Server.CurrentInstance.SessionManager.SessionCreated += EventStatus;
```

Comment:
[Improvement] not related to reverse connect -  deregistration from events is missing